doc/tutorials/klondike/app/lib/step5/klondike_game.dart

```diff
@@ -22,6 +22,9 @@ class KlondikeGame extends FlameGame<KlondikeWorld> {
     const Radius.circular(cardRadius),
   );
 
+  // Constant used to decide when a short drag will be treated as a TapUp event.
```

Comment:
```suggestion
  /// Constant used to decide when a short drag will be treated as a TapUp event.
```
Comments that are explaining public variables or classes can be dartdocs.